### PR TITLE
Use test kit in sample UI tests

### PR DESCRIPTION
**Pull request by khalp**
Update samples with UI tests to use test kit spanning methods instead of local helper methods

Note: some of the UI tests in `FoldingVideo` and `PhotoEditor` fail but for reasons other than this update - I can create issues for these so we can fix them in the future

### CODE REVIEW

**Review by conceptdev — APPROVED** [review]
Code definitely looks neater!

Do we know what caused the failing `FoldingVideo` and `PhotoEditor` tests?

**Comment by khalp** [comment]
> Code definitely looks neater!
> 
> Do we know what caused the failing `FoldingVideo` and `PhotoEditor` tests?

Glad you think so 😄 

For `PhotoEditor`, the drag and drop test is failing because I think we're using an outdated method for starting a different app (the Files app). `FoldingVideo` might need a little more investigation, but I think it might be related to the animated entrance of the controls

File: PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/wm_samples/photoeditor/PhotoEditorUITest.kt [hunk 1]
```diff
@@ -122,90 +126,9 @@ class PhotoEditorUITest {
...
-        device.swipe(leftX, bottomY, rightX, middleY, switchSteps)
-    }
-
-    private fun closeLeft() {
-        device.swipe(leftX, bottomY, leftX, middleY, closeSteps)
-    }
-
-    private fun closeRight() {
-        device.swipe(rightX, bottomY, rightX, middleY, closeSteps)
-    }
```
Comment (CesarValiente) [inline PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/wm_samples/photoeditor/PhotoEditorUITest.kt:210]: I 💖 the smell of deleted code 🔝😄

File: TwoNote/ink/build.gradle [hunk 2]
```diff
@@ -22,7 +22,6 @@ android {
     defaultConfig {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionName rootProject.ext.inkSDKVersionName
```
Comment (CesarValiente) [inline TwoNote/ink/build.gradle:25]: good catch!

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks for doing this, Kristen! 👏👏

**Pull request merged**